include/wx/webview.h

```diff
@@ -223,6 +223,10 @@ class WXDLLIMPEXP_WEBVIEW wxWebView : public wxControl
 protected:
     virtual void DoSetPage(const wxString& html, const wxString& baseUrl) = 0;
 
+    //Counts number of calls of RunScript, to avoid
```

Comment:
A better wording would be "Count _the_ number of calls _to_ RunScript() in order to prevent _the_same_ variable _from_ being used twice _in_ more than one call."